libsrc/Makefile

```diff
@@ -14,14 +14,16 @@ CBMS = c128   \
 GEOS = geos-apple \
        geos-cbm
 
-TARGETS = apple2    \
+TARGETS = \
+          $(CBMS)   \
+          $(GEOS)   \
+          apple2    \
           apple2enh \
           atari     \
-          atarixl   \
           atari5200 \
+          atarixl   \
```

Comment:
Greg told me that this list must be in ascending order..


On 10.05.2015 13:51, Oliver Schmidt wrote:
>
> In libsrc/Makefile 
> <https://github.com/cc65/cc65/pull/157#discussion_r30001365>:
>
> >            atari5200 \
> > +          atarixl   \
>
> This pull request isn't related to Atari stuff so please leave Atari 
> stuff alone. I strongly disgree to mix "general cleanup" operations 
> with significant changes. Apart from that the order of Atari items was 
> carefully chosen.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/cc65/cc65/pull/157/files#r30001365>.
>

-- 
Mit freundlichen Grüssen

Rainer Keuchel